ovirt/resource_ovirt_vm.go

```diff
@@ -174,6 +174,11 @@ func resourceOvirtVM() *schema.Resource {
 							Required: true,
 							ForceNew: true,
 						},
+                                                "alias": {
+                                                        Type:     schema.TypeString,
+                                                        Optional: true,
+                                                        ForceNew: false,
+                                                },
```

Comment:
Please squash the next formatting commit with this one.

Reply:
Done